pkg/controller/allowlist/allowlist_controller.go

```diff
@@ -76,6 +97,7 @@ func (r *ReconcileAllowlist) Reconcile(ctx context.Context, request reconcile.Re
 	if request.Namespace != names.MULTUS_NAMESPACE || request.Name != names.ALLOWLIST_CONFIG_NAME {
 		return reconcile.Result{}, nil
 	}
+	log.Printf("Reconcile allowlist for %s/%s", request.Namespace, request.Name)
```

Comment:
please use klog to keep consistency on this file

Reply:
Ok, will do. I had been following code of the other controllers and didn't notice this controller was using klog.

Reply:
Done